cmake/kokkos_enable_options.cmake

```diff
@@ -52,7 +52,11 @@ KOKKOS_ENABLE_OPTION(EXPLICIT_INSTANTIATION   OFF
   "Whether to explicitly instantiate certain types to lower future compile times")
 SET(KOKKOS_ENABLE_ETI ${KOKKOS_ENABLE_EXPLICIT_INSTANTIATION} CACHE INTERNAL "eti")
 
-IF (DEFINED CUDA_VERSION AND CUDA_VERSION VERSION_GREATER "7.0")
+IF (KOKKOS_ENABLE_CUDA)
+  SET(KOKKOS_COMPILER_CUDA_VERSION "${KOKKOS_VERSION_MAJOR}${KOKKOS_VERSION_MINOR}")
+ENDIF()
+
+IF (DEFINED KOKKOS_COMPILER_CUDA_VERSION AND KOKKOS_COMPILER_CUDA_VERSION GREATER 70)
   SET(LAMBDA_DEFAULT ON)
```

Comment:
Nope, I am against this. As long as NVIDIA doesn't officially support this I would say no. I.e. I want them to make it a non-experimental feature first.

Reply:
OK, I will change that.